index.html

```diff
@@ -8,6 +8,6 @@
     <title>Hello World</title>
```

Comment:
## 2
### Your Summary
![Screenshot from 2022-09-27 18-58-16](https://user-images.githubusercontent.com/93039359/192549403-c9a88285-8723-49d1-8a5a-232a3567faa5.png)


### Descriptive Summary Example :heavy_check_mark:
![Screenshot from 2022-09-27 19-11-39](https://user-images.githubusercontent.com/93039359/192550220-b7fb8f35-36b1-4db0-bc07-adb07569322b.png)


- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: